add-domain

```diff
@@ -747,3 +747,11 @@ soclaiemx.xyz
 wiflix.xyz
 wiflix.yachts
 coronavirus.zone
+xss.ls
+elektrum.sbs
+www.electrum.la
+electrum.la
+v2-electrum.com
+elektrum.icu
+elektrum.cfd
+web3-electrum.com
```

Comment:
Working, but not convinced it is phishing

![image](https://github.com/user-attachments/assets/87e52fd5-df57-4ab3-9c7f-e6d18164775e)

Reply:
Please check the detection score. All of them are used for phishing. They are even flagged by Microsoft Defened SmartScreen, Google Safebrowsing and Spamhaus.

elektrum[.]icu & elektrum[.]cfd has been taken down.

1. **https://check.spamhaus.org/results/?query=www.electrum.la**
2. Microsoft Defender SmartScreen
3. https://radar.cloudflare.com/domains/domain/electrum.la
4. https://www.virustotal.com/gui/domain/electrum.la
5. https://urlfiltering.paloaltonetworks.com/single_cr/?url=electrum.la
6. https://www.quad9.net/result/?url=electrum.la
7. https://exchange.xforce.ibmcloud.com/url/electrum.la
8. https://talosintelligence.com/reputation_center/lookup?search=electrum.la
9. https://safeweb.norton.com/report/show?url=electrum.la
10. https://maltiverse.com/hostname/www.electrum.la
11. https://reports.adguard.com/en/electrum.la/report.html

1. **https://check.spamhaus.org/results?query=web3-electrum.com**
2. **https://check.spamhaus.org/results?query=v2-electrum.com**
3. https://safeweb.norton.com/report/show?url=v2-electrum.com
4. https://www.virustotal.com/gui/domain/v2-electrum.com
5. https://www.virustotal.com/gui/domain/web3-electrum.com
6. https://safeweb.norton.com/report/show?url=web3-electrum.com

![msedge_G7RYC04fWq](https://github.com/user-attachments/assets/2660c330-0350-435b-9bd9-d614198d1917)